lib/applyResponsiveStyleModifiers.js

```diff
@@ -11,10 +18,12 @@ export default function applyResponsiveStyleModifiers(
   modifierConfig,
   modifierPropName = 'responsiveModifiers',
 ) {
-  return props => {
+  return (props) => {
```

Comment:
I thought our lint rules discouraged the use of parens around a single function argument with the arrow syntax?

Reply:
It does, but it is weakly enforced right now. I think @seanmpage's PR will include prettier and will help standardize all this.